components/RoundButton.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <a :href="url">
+    <button class="RoundButton">{{ name }}</button>
+  </a>
+</template>
+
+<script>
+export default {
+  // eslint-disable-next-line vue/require-prop-types
+  props: ['name', 'url']
```

Comment:
こちら、最低限Type設定をしてほしいです。
```suggestion
  props: {
    name: {
      type: String,
      required: true
    },
    url: {
      type: String,
      required: true
    }
  }
```